scripts/run_e2e_tests.py

```diff
@@ -486,6 +486,27 @@ def run_e2e_tests(
                         ["streamlit", "run", "--ui.hideSidebarNav=false", test_path],
                         show_output=verbose,
                     )
+            elif basename(spec_path) == "staticfiles_app.spec.js":
+                test_name, _ = splitext(basename(spec_path))
+                test_name, _ = splitext(test_name)
+                test_path = join(
+                    ctx.tests_dir,
+                    "scripts",
+                    "staticfiles_apps",
+                    "streamlit_static_app.py",
+                )
+                if os.path.exists(test_path):
```

Comment:
Why is this `os.path.exists` check here? Do we expect the path to ever not exist? Shouldn't that be a failure?

Reply:
I think we don't expect missing test files, but I added this check just because they also exist in [corresponding blocks for `multipage_apps`](https://github.com/streamlit/streamlit/blob/15ff5118f06c7bec4f004e0ff0483d2232819b2e/scripts/run_e2e_tests.py#L482) , and in [`else` block where we run test files by default](https://github.com/streamlit/streamlit/blob/15ff5118f06c7bec4f004e0ff0483d2232819b2e/scripts/run_e2e_tests.py#L494).